tests/post_training/data/wc_reference_data.yaml

```diff
@@ -32,11 +32,11 @@ tinyllama_data_aware_gptq_scale_estimation_stateful_backend_OV:
   num_int8: 124
   metrics_xfail_reason: "Issue-148819"
 tinyllama_scale_estimation_per_channel_backend_OV:
-  metric_value: 0.81389
+  metric_value: 0.80873
   num_int4: 188
   num_int8: 124
 tinyllama_scale_estimation_per_channel_backend_TORCH:
-  metric_value: 0.81389
+  metric_value: 0.80873
   num_int4: 188
   num_int8: 124
   atol: 0.006 # difference across devices: 0.80873 vs 0.81389
```

Comment:
A reference value for PT is changed to stay aligned with OV. Interestingly, the new value for OV (0.80873) is also the value that PT backend gets on some devices.